src/TritonCTS/CMakeLists.txt

```diff
@@ -26,8 +26,20 @@ add_custom_command(OUTPUT ${TRITONCTS_TCL_INIT}
         WORKING_DIRECTORY ${TRITONCTS_HOME}/src
         DEPENDS ${TRITONCTS_HOME}/src/tritoncts.tcl ${OPENSTA_HOME}/etc/TclEncode.tcl
 )
+set(SRC_FILES
+        ${CMAKE_CURRENT_SOURCE_DIR}/src/Clock.cpp
```

Comment:
I don't think you need to prefix the lines with ${CMAKE_CURRENT_SOURCE_DIR}.
If you move the cmake file down to /src you won't need the src/ either (which is what
resizer does)

Reply:
I replicated the resizer topology (which has two CMake files) and removed the ${CMAKE_CURRENT_SOURCE_DIR}/src occurrences.